reaktive/src/nativeCommonMain/kotlin/com/badoo/reaktive/utils/atomic/AtomicBoolean.kt

```diff
@@ -0,0 +1,20 @@
+package com.badoo.reaktive.utils.atomic
+
+actual class AtomicBoolean actual constructor(initialValue: Boolean) {
+
+    private val delegate = kotlin.native.concurrent.AtomicInt(initialValue.intValue)
+
+    actual var value: Boolean
+        get() = delegate.value != 0
+        set(value) {
+            delegate.value = value.intValue
+        }
+
+
+    actual fun compareAndSet(expectedValue: Boolean, newValue: Boolean): Boolean =
+        delegate.compareAndSet(expectedValue.intValue, newValue.intValue)
```

Comment:
Make it single line as in other implementations.

Reply:
It exceeds 140 chars then, and wrapped by auto-format